tests/unittests/test_datasource/test_azure.py

```diff
@@ -392,6 +392,9 @@ def setUp(self):
         # patch cloud_dir, so our 'seed_dir' is guaranteed empty
         self.paths = helpers.Paths(
             {'cloud_dir': self.tmp, 'run_dir': self.tmp})
+        write_file(
```

Comment:
@AOhassan can we please drop this write_file here, It should only be needed for this single unit tests  where you are testing byte swapped vs not. test_instance_id_endianness